features/old/accounts/buyers/personal_details.feature

```diff
@@ -39,8 +39,6 @@ Feature: Personal Details
     And I follow "Settings"
     Then I should not see "Personal Details"
 
-
-  @javascript
```

Comment:
It's asserting the `*` is present (to identify required fields) and it's hidden by JS. Since now capybara ignores hidden elements, it fails. I'll fix it.